### DISCUSSION TITLE
ElasticSearch Showing Mapping Error. I had created Index and Aliase and done the mapping and hit the search API it showing this error

### DISCUSSION THREAD

**Discussion by Tusharmahajan12** [Sunbird-Ed/Community#396] [Sunbird ED configuration discussion] [answered]
`  "errmsg": "method [POST], host [http://localhost:9200], URI [/compositesearch/_search?rest_total_hits_as_int=true&typed_keys=true&ignore_unavailable=false&expand_wildcards=open&allow_no_indices=true&ignore_throttled=true&search_type=query_then_fetch&batched_reduce_size=512], status line [HTTP/1.1 400 Bad Request]\n{\"error\":{\"root_cause\":[{\"type\":\"query_shard_exception\",\"reason\":\"No mapping found for [name.raw] in order to sort on\",\"index_uuid\":\"6LEHuAj1Sy24oL_ue_sthg\",\"index\":\"compositesearch_v1\"}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"compositesearch_v1\",\"node\":\"1s-LwausRmOWrsdo2UrByw\",\"reason\":{\"type\":\"query_shard_exception\",\"reason\":\"No mapping found for [name.raw] in order to sort on\",\"index_uuid\":\"6LEHuAj1Sy24oL_ue_sthg\",\"index\":\"compositesearch_v1\"}}]},\"status\":400}"`

**Comment by gandham-santhosh** [comment, marked as answer]
@Tusharmahajan12 can you please share the following details?
- Are you trying to setup SunbirdEd in CSP? If so which version of Ed you are trying to setup? Which cloud service provider you are using?
-  How did you create the index? If it is using Jenkins, please share the Jenkins job path
- How are you calling searchAPI? Please share the API details with body

**Reply by Tusharmahajan12** [reply]
@gandham-santhosh 
I had setup on my local machie SunbirdEd Knowledge-Platform version 5.1.0.
Using Curl command i had created index. No jenkins.
I am having routes which is running on search port http://localhost:9020/v3/search. the body 
`{
  "request": {
    "filters": {
      "objectType": "Content",
      "status": []
    },
    "limit": 1
  }
}`

**Reply by gandham-santhosh** [reply]
@Tusharmahajan12  as this is specific to knowlg repo local setup, can you please have your query added to Knowlg BB discussions at https://github.com/orgs/Sunbird-Knowlg/discussions ?

**Reply by chitranshu-keshav** [reply]
@Tusharmahajan12 Can you please share the link of discussion thread created in Knowlg discussion forum and mark this thread as answered here

**Reply by Tusharmahajan12** [reply]
https://github.com/orgs/Sunbird-Knowlg/discussions/94
